_posts/2019/2019-09-21-internal-hackathon-arcore-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+ layout: post
+ title: Working with ARCore
+ excerpt: Playing with Unity and Google’s ARCore
+ author:
+   name: Shubham Gupta
+   link: https://github.com/shubhamgupta2956
+   bio: Developer, SDSLabs
+   image: shubham.jpg
+---
+
+# The Motivation behind this blog
+
+Installing Unity is an easy task. Just download the setup files and after accepting all the agrements and defining the various paths,etc. , it will install automatically and you are good to go. But the main problem comes when you want to do android development using unity.
+
+# What is ARCore?
+
+According to Wikipedia, ARCore is a software development kit developed by Google that allows for augmented reality applications to be built.
+ARCore uses three key technologies to integrate virtual content with the real environment:
+* Motion Tracking: it allows the phone to understand its position relative to the world.
+* Environmental understanding: It allows the phone to detect the size and location of all type of surfaces, vertical, horizontal and angled.
+* Light Estimation: it allows the phone to estimate the environment’s current lighting conditions.
+
+# Setting up development environment
+
+## Requirements
+
+*Unity 2017.4.26f1 or later
+
+**Make sure to include Android Build Support during installation
+
+**The Universal Render Pipeline (formerly known as Ligthweight Render Pipeline or LWRP) is not supported by the ARCore SDK for Unity
+
+**When using Unity 2019, the following Unity packages are required:
+
+1. Multiplayer HLAPI
```

Comment:
Make things uniform. Having numerical list here and bulleted elsewhere makes the hierarchy confusing